ldk/javascript/src/webpack/generate-banner.test.ts

```diff
@@ -27,6 +27,7 @@ describe('Generate Banner', () => {
         keyboard: {},
         network: { urlDomains: [{ value: '*.google.com' }] },
         process: {},
+        search: {},
```

Comment:
You probably know this better than me, but I believe a change in metadata in the LDK will also require a schema change within LoopLibrary.

Reply:
Yes, you are correct, that is on the list of things we are planning on still getting done